pages/about.html

```diff
@@ -0,0 +1,142 @@
+        </div>
+    
+        <div class="top-nav-2">
+            <a href="#">English</a>
+            <a href="#">My Page</a>
+            <a href="#">Logout</a>
+        </div>
+      </div>
+        
+      <header>
+        <nav>
+            <a href="/index.html">
+            <div id="header-logo">
+              <img class="header-logo-image" src="/images/logo.png" alt="logo">
+              <p> GSSO Alumni Ass.</p>
+            </div></a>
+            <div id="nav-items" class="nav-items">
+              <a href="/pages/about.html">About</a>
+              <a href="#">Program</a>
+              <a href="#">Join</a>
+              <a href="#">Sponsor</a>
+              <a href="#">News</a>
+              <a href="#">CC Campaign</a>
+            </div>
+    
+            <button type="button" id="menu-btn" class="menu-btn">
+              <i class="fas fa-bars"></i>
+            </button>
+        
+            <button type="button" class="close-btn" id="close-btn">
+              <i class="fa-solid fa-xmark"></i>
+            </button>
+        </nav>
+      </header>
+        
+    <section id="about" class="about">
+        <div class="about-content">
+            <div class="about-title">
+                <h2>"Great GSSO 2006 Set"</h2>
+              <h1>GOVERNMENT SECONDARY SCHOOL OLD BOYS HOME COMING</h1>
+            </div>
+            <div class="about-description">
+              <p>The Global Summit brings together the community of 
+                experts academics, and activitists who comprise the 
+                Creative Commons affiliate network in diferent country
+                every two. This year, we hope to extend our invitation 
+                list including organisations and individuals</p>
+            </div>
+            <div class="about-contact">
+              <p>Please contact us by Email for any further questions
+                 about GSSO Home Coming 2023</p>
+              <h3>gsso.org</h3>
+            </div>
+
+            <div class="form-container">
+            <form id="form"  action="https://formspree.io/f/xqkjanrn"  method="POST">
+                <fieldset>
+                    <legend>Send me a quick message</legend>
+                    Your Name:
+                    <input type="text" name="fullname" maxlength="30" placeholder="Fullname" required>
+                <label>
+                    Your email:
+                    <input id="mail" class="email" type="email" name="email" placeholder="Email address">
+                  </label>
+                    <label>
+                        Your message:  
+                        <textarea name="comment" required placeholder="Type your comment here" cols="30" rows="10"></textarea>
+                    </label>
+                    <span class="message"></span>
+                    <!-- your other form fields go here -->
+                    <button type="submit">Send</button>
+                </fieldset>
+            </form>
```

Comment:
[OPTIONAL] It's better to remove this form since this is not the part of design 🤗